prow/monitoring/BUILD.bazel

```diff
@@ -13,6 +13,12 @@ release(
     component("prow", "prometheus"),
     component("prometheus_expose", MULTI_KIND),
     component("prometheus", "servicemonitor"),
+    component("prow", "prometheusrule"),
+    #kubectl create -n prow-monitoring secret generic alertmanager-prow --from-file=alertmanager.yaml=./alertmanager.yaml --from-file=./msg.tmpl --dry-run -o yaml | kubectl apply -f -
```

Comment:
https://kubernetes.slack.com/archives/CDECRSC5U/p1560274782066500?thread_ts=1560274627.064100&cid=CDECRSC5U
2 blockers:

* `k8s_secret` is not supported yet
* need a way to inject the sensitive info:

https://github.com/kubernetes/test-infra/blob/2fc7194878cb1843b7f4b9af573cf77a32d0227d/prow/monitoring/alertmanager.yaml#L15

Reply:
```
#kubectl create -n prow-monitoring secret generic alertmanager-prow --from-file=alertmanager.yaml=./alertmanager.yaml --from-file=./msg.tmpl --dry-run -o yaml | kubectl apply -f -
```
Use a workaround for `k8s_secret`:

https://github.com/kubernetes/test-infra/blob/2cef06c2023b21d002a9a3b37fab2707e3dce8d3/prow/monitoring/BUILD.bazel#L18

We will figure out the solution to the 2nd blocker in the future.